### DISCUSSION TITLE
我设置了仅开启 gemini1.5Pro 他还是会显示别的

### DISCUSSION THREAD

**Discussion by ghost** [lobehub/lobehub#1741] [General Question] [answered]
### 💻 系统环境

Windows

### 📦 部署环境

Vercel / Zeabur / Sealos

### 🌐 浏览器

Edge

### 🐛 问题描述

我设置了仅开启gemini1.5Pro他还是会显示别的，我设置的-all,+gemini-1.5-pro，里面显示别的
![Screenshot_20240323_161611](https://github.com/lobehub/lobe-chat/assets/138294823/a06c9007-4f0d-4938-91b8-29ce8684d914)

[a comment by an automated account is not shown]

**Comment by CabbSir** [comment, marked as answer]
应该是CUSTOM_MODELS这个环境变量是针对OPENAI服务商的。然后你应该又填了Google的环境变量，就这样了。

[a comment by an automated account is not shown]

**Comment by MapleEve** [comment]
这个 Gemini 1.5 并不是 1.5

[a comment by an automated account is not shown]
